resources/views/groups/edit-group.blade.php

```diff
@@ -61,15 +61,14 @@
                                 </div>
 
                                 <div class="col-5">
-                                    <!-- Second column -->
-                                    <div class="row row-cols-2">
-                                        <div class="row" style="width: 40%">
-                                            <img src="{{ $group[0]->photo }}" class="rounded-circle d-flex px-0" style="width: 160px; height: 160px;"
-                                                 id="img_group" alt="Avatar"/>
+                                    <!-- Second column -->sssssssssssssssssssssssssss
```

Comment:
the "ssss..," text is probably unintentional and certainly unwanted

Reply:
otherwise everything looks good to me

Reply:
Oh right, thank you 👍 I will fix it. 😉